includes/modules/order_total/ot_coupon.php

```diff
@@ -57,7 +57,11 @@ function ot_coupon() {
   function process() {
     global $order, $currencies;
     $order_total = $this->get_order_total(isset($_SESSION['cc_id']) ? $_SESSION['cc_id'] : '');
-    $od_amount = $this->calculate_deductions();
+    if ($order_total > 0) { 
+       $od_amount = $this->calculate_deductions();
+    } else { 
+       $od_amount = array('tax'=>0, 'total'=>0);
+    }

```

Comment:
Rather than use an `else` at all, what do you think of first initializing `$od_amount` to the zero-array before the `if`?